app/_hub/kong-inc/grpc-web/index.md

```diff
@@ -0,0 +1,158 @@
+---
+name: gRPC-Web
+publisher: Kong Inc.
+
+categories:
+  - transformations
+
+type: plugin
+
+desc: Allow browser clients to call gRCP services.
+description: |
+  A Kong plugin to allow access to a gRPC service via the gRPC-Web protocol.
+  Primarily, this means JS browser apps using the gRPC-Web library.
+  
+source_url: https://github.com/Kong/kong-plugin-grpc-web
+
+license_type: MIT
+
+kong_version_compatibility:
+  community_edition:
+    compatible:
+      - 2.0.x
+      - 1.5.x
+      - 1.4.x
+
+
+params:
+  name: grpc-web
+  route_id: true
+  protocols: ["http", "https"]
+  dbless_compatible: true
+
+  config:
+    - name: proto
+      required: false
+      default:
+      value_in_examples: path/to/hello.proto
+      description: |
+        If present, describes the gRPC types and methods.
+        Required to support payload transcoding.  When absent, the
+        web client must use application/grpw-web+proto content.
+
+---
+
+## Purpose
+
+A service that presents a gRPC API can be used by clients written in many languages, but the network specifications are oriented primarily to connections within a datacenter.  In order to expose the API to the Internet, and to be called from brower-based JS apps, [gRPC-Web] was developed.
+
+This plugin translates requests and responses between gRPC-Web and "real" gRPC.  Supports both HTTP/1.1 and HTTP/2, over plaintext (HTTP) and TLS (HTTPS) connections.
+
+## Usage
+
+This plugin is intended to be used in a Kong route between a gRPC service and an HTTP endpoint.
```

Comment:
```suggestion
This plugin is intended to be used in a Kong route between an HTTP endpoint and a gRPC service.
```

Very minor/stylistic, but I think ordering by "source, then destination" makes it easier to read/understand.